airflow/providers/common/sql/CHANGELOG.rst

```diff
@@ -25,6 +25,14 @@
 Changelog
 ---------
 
+1.12.0
+......
+
+Features
+~~~~~~~~
+
+* ``New generic SQL tranfer opertor: SqlToSqlOperator``
+
```

Comment:
This is not needed. The log is auto generated 
```suggestion

```

Reply:
I commited change